resources/js/Components/SortDropdown.tsx

```diff
@@ -0,0 +1,74 @@
+import cn from "classnames";
+import { type ReactElement, type ReactNode } from "react";
+import { useTranslation } from "react-i18next";
+import { IconButton } from "@/Components/Buttons";
+import { Dropdown } from "@/Components/Dropdown";
+import { Tooltip } from "@/Components/Tooltip";
+
+interface Properties {
+    disabled?: boolean;
+    children: ReactElement | (({ setOpen }: { setOpen: (open: boolean) => void }) => JSX.Element | JSX.Element[]);
+}
+
+export const SortDropdown = ({ disabled = false, children }: Properties): JSX.Element => {
```

Comment:
do we need a new component for this or could we reuse the existing stuff from collections for it?

Reply:
Yes, we need that component, I added intentionally to make it reusable: 

Checkout how it is being used in the collections page: https://github.com/ArdentHQ/dashbrd/blob/develop/resources/js/Pages/Collections/Components/CollectionsFilter/CollectionsSorting.tsx

Proposed solution: https://github.com/ArdentHQ/dashbrd/pull/158/files#diff-ee484fe7ed5066defacf979c111321ab58b37534f196df692fbc0d8d8683d3cdR18

Basically, we can refactor the collections page to reuse this component.